devicetypes/smartthings/aeon-multisensor-6.src/aeon-multisensor-6.groovy

```diff
@@ -442,10 +442,7 @@ def configure() {
 
 	// set the check interval based on the report interval preference. (default 122 minutes)
 	// we do this here in case the device is in wakeup mode
-	def checkInterval = 2 * 60 * 60 + 2 * 60
-	if (reportInterval) {
-		checkInterval = 2 * timeOptionValueMap[reportInterval] + (2 * 60)
-	}
+	def checkInterval = 2 * (timeOptionValueMap[reportInterval] ?: 60 * 60) + 2 * 60
```

Comment:
@greens any idea why the default previously used here isn't the same as what's used here: https://github.com/SmartThingsCommunity/SmartThingsPublic/blob/732a0287331fa208854a195357c77e6d436827f2/devicetypes/smartthings/aeon-multisensor-6.src/aeon-multisensor-6.groovy#L412?

Reply:
no reason other than 8 minutes is a pretty short duration to fallback to

Reply:
I feel like these values should probably be the same but for now I'm going to leave it as is and at least get this fix out.